### PR TITLE
Expose node exec options

### PR DESCRIPTION
**Pull request by tadam313**
Exposed `exec` option via webpack loader `option` object.  **Use case**: pass `maxBuffer` property to exec call in order to handle very large input files.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 const loaderUtils = require('loader-utils');
+const omit = require('lodash.omit');
 
 module.exports = function(source) {
-    var callback = this.async();
-    var options = loaderUtils.getOptions(this);
-    var command = exec(options.script, function(err, result) {
+    const options = loaderUtils.getOptions(this);
+    const callback = this.async();
+    const scriptOptions = omit(options, 'script');
```
Comment (localjo) [inline index.js]: I like this idea. Could you do this without using lodash? I don't want to have unneeded dependencies. Maybe something like;

```
const { script, ...options } = loaderUtils.getOptions(this);
exec(script, options, function() {
  // 
})
```

I didn't test this, and this project isn't fresh enough in my mind to remember if ES6 features will work here.

**Review by localjo — COMMENTED** [review]
Thanks for the PR. This looks interesting. Out of curiosity, what are the use cases for this?

File: index.js (hunk 1 above)
Comment (tadam313) [reply index.js]: ok, sure if we can use ES6 then no problem :) wasn't sure if you wanted to maintain compatibility with very old node versions (<=4).

**Comment by tadam313** [comment]
@localjo made the requested changes. **Use case**: we have very large files which we wanna pass through a custom rails transformer script via webpack (due to migrating away from rails). Unfortunately the files larger than `exec` default buffer size so we had to increase it.

**Comment by tadam313** [comment]
@localjo can you please review it again? I've made the changes

File: index.js [hunk 2]
```diff
@@ -2,12 +2,14 @@ const { exec } = require('child_process');
 const loaderUtils = require('loader-utils');
 
 module.exports = function(source) {
-    var callback = this.async();
-    var options = loaderUtils.getOptions(this);
-    var command = exec(options.script, function(err, result) {
+    const { script, ...options } = loaderUtils.getOptions(this);
+    const callback = this.async();
```
Comment (localjo) [inline index.js:5]: Just a small nitpick: can you put this line back at the top of the function and remove the extra whitespace below and on line 12?

Comment (tadam313) [reply index.js:5]: sure, it's done!

**Comment by localjo** [comment]
Thanks @tadam313 merged and released version 1.2.0 on npm

**Pull request merged**